scripts/lib/CIME/build.py

```diff
@@ -353,7 +354,7 @@ def _build_libraries(case, exeroot, sharedpath, caseroot, cimeroot, libroot, lid
     cdeps_build_script = None
     if comp_interface == "nuopc":
         libs.append("CDEPS")
-        cdeps_build_script = os.path.join(cimeroot, "src", "components", "cdeps", "cime_config", "buildlib")
+        cdeps_build_script = os.path.join(srcroot, "components", "cdeps", "cime_config", "buildlib")
```

Comment:
This should use config_files and not assume a path.

Reply:
I am fine doing that - but this was not done in the original implementation so can we please discuss how we waht to have this implemented. I simply changed a path that had already been hard-wired.